### PR TITLE
[ADD] l10n_ch_mis_reports: add new module

### PR DESCRIPTION
**Pull request by mpanarin**
add mis report data for Switzerland localization

### CODE REVIEW

**Review by yvaucher — APPROVED** [review]

**Comment by yvaucher** [comment]
Code is ok but a README.rst is missing.

**Comment by fclementic2c** [comment]
Please remove the ' (at the beguining before the formula) on expressions
example on line  <field name="expression">'-bale[][('account_id.tag_ids', '=', ref('l10n_ch.account_tag_ch_280').id)]</field>

**Comment by yvaucher** [comment]
@fclementic2c will do, this char was probably inserted on the export from Odoo

**Comment by yvaucher** [comment]
@fclementic2c extra `'` removed from xml file.

**Comment by fclementic2c** [comment]
Please also remove ' on line description (see printscreen) - This is pure cosmetic. txs

screenshot-localhost-8989-2018 06 21-13-24-33 1

**Comment by yvaucher** [comment]
@fclementic2c already done in last commit:
https://github.com/OCA/l10n-switzerland/pull/429/commits/7ee8a6e61cafeaebca47cc37a018727cd8b1fd0e#diff-00fed281fc62b3a8a85796e6ef8bfa6cL99

**Comment by fclementic2c** [comment]
Rest is Ok :)

Good Job !

**Comment by yvaucher** [comment]
@mpanarin can you do a rebase on top of https://github.com/OCA/l10n-switzerland/pull/432 as this depends on it

File: l10n_ch_mis_reports/__manifest__.py
Comment (yvaucher) [inline l10n_ch_mis_reports/__manifest__.py:15]: Add dependency on `l10n_ch_account_tags`

**Comment by mpanarin** [comment]
@yvaucher rebase and updated

File: l10n_ch_mis_reports/__manifest__.py
Comment (yvaucher) [reply l10n_ch_mis_reports/__manifest__.py:15]: ok

**Comment by yvaucher** [comment]
@mpanarin thanks, waiting for travis before merge

**Pull request merged**
